G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelAddSheet/ShouldAddSheet_WhenExcelAddSheetCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,14 @@
+﻿excel.open
+
+for counter ♥a from 1 to 3 step 1
+excel.addsheet name ‴TestSheet♥a‴
+excel.setvalue value ♥a row 1 colindex 1
+end
+
+for counter ♥counter from 1 to 3 step 1
+    excel.activatesheet name ‴TestSheet♥counter‴
+    excel.getvalue row 1 colindex 1 result ♥cellValue
+    test ⊂♥cellValue == ♥counter.ToString()⊃
```

Comment:
Don't call `test` before `excel.close`, otherwise an instance of excel will remain open across the other tests.